ui/00_message/skill/custom_skill_command_03.xml

```diff
@@ -61,11 +61,12 @@
   </message>
   <message>
     <key>CUSTOM_SKILL_COMMAND_03_131</key>
-    <original>セットボタンを押す
-</original>
```

Comment:
That's fine, just pointing it out. My tooling requires original text to be left alone for things like merging in patch changes, etc.